Adding post-processing folder to store scripts used to post-process data

lucbv opened the pull request:
Currently it contains a module that parses the log files from Nalu
It extracts data per equation as well as some global data.
Also adding a script showing how to use the module to generate
FY20Q2 milestone weak scaling tables.

sayerhs commented on post-processing/post_process_nalu.py:
```diff
@@ -0,0 +1,111 @@
+    times       = np.zeros((len(equations), 8))
+    for lineIdx in range(len(data)):
+        line = data[lineIdx]
+	if("STKPERF: Total Time:" in line):
```
Looks like wrong indentation.

lucbv replied:
it's actually a tab vs whitespace issue, I will run a command that makes everything whitespaces.

lucbv committed:
Adding post-processing folder to store scripts used to post-process data

Currently it contains a module that parses the log files from Nalu
It extracts data per equation as well as some global data.
Also adding a script showing how to use the module to generate
FY20Q2 milestone weak scaling tables.

lucbv commented:
@sayerhs how about know, I ran untabify, it should be OK

lucbv committed:
Fixing tab vs whitespace issue

sayerhs reviewed (approved):
lgtm

lucbv committed:
Adding table headers
